pyplanet/core/db/database.py

```diff
@@ -41,14 +41,7 @@ def __init__(self, engine_cls, instance, *args, **kwargs):
 	@classmethod
 	def create_from_settings(cls, instance, conf):
 		try:
-			engine_path, _, cls_name = conf['ENGINE'].rpartition('.')
-			db_name = conf['NAME']
-			db_options = conf['OPTIONS'] if 'OPTIONS' in conf and conf['OPTIONS'] else dict()
-
-			# FIX for #331. Replace utf8 by utf8mb4 in the mysql driver encoding.
-			if conf['ENGINE'] == 'peewee_async.MySQLDatabase' and 'charset' in db_options and db_options['charset'] == 'utf8':
-				logging.info('Forcing to use \'utf8mb4\' instead of \'utf8\' for the MySQL charset option! (Fix #331).')
-				db_options['charset'] = 'utf8mb4'
```

Comment:
The charset no longer needs to be forced?

Reply:
I set the default to be this.